lib/testrail.js

```diff
@@ -24,102 +23,131 @@ class TestRail {
+			output.error(`addRun: ERROR - request data was ${JSON.stringify(data)}`);
 		}
 	}
 
 	async updateRun(runId, data) {
 		try {
 			const res = await this.axios.post('update_run/' + runId, data);
-			output.log(`The run with id: ${runId} is updated`);
+			output.log(`updateRun: SUCCESS - the request data is ${JSON.stringify(data)}`);
+			output.log(`updateRun: SUCCESS - the response data is ${JSON.stringify(res.data)}`);
 			return res.data;
 		} catch (error) {
 			const parsedError = error && error.response && error.response.data ? error.response.data.error : error;
-			output.error(`Cannot update run due to ${parsedError}`);
-			output.error(`Request data was: ${JSON.stringify(data)}`);
+			output.error(`updateRun: ERROR - cannot get results for case ${caseId} on run ${runId} due to ${parsedError}`);
+			output.error(`updateRun: ERROR - request data was ${JSON.stringify(data)}`);
 		}
 	}
 
 	async getResultsForCase(runId, caseId) {
 		return this.axios.get(`get_results_for_case/${runId}/${caseId}`).then((res) => {
-			output.log(`The response is ${JSON.stringify(res.data)}`);
-			output.log(`The case ${caseId} on run ${runId} is updated`);
+			output.log(`getResultsForCase: SUCCESS - the request data is ${JSON.stringify(data)}`);
+			output.log(`getResultsForCase: SUCCESS - the response data is ${JSON.stringify(res.data)}`);
 			return res.data;
 		}).catch(error => {
-			output.error(`Cannot get results for case ${caseId} on run ${runId} due to ${error}`);
+			const parsedError = error && error.response && error.response.data ? error.response.data.error : error;
+			output.error(`getResultsForCase: ERROR - cannot get results for case ${caseId} on run ${runId} due to ${parsedError}`);
+			output.error(`getResultsForCase: ERROR - request data was ${JSON.stringify(data)}`);
 		});
 	}
 
 	async addResultsForCases(runId, data) {
 		return this.axios.post('add_results_for_cases/' + runId, data).then((res) => {
-			output.log(`The response is ${JSON.stringify(res.data)}`);
+			output.log(`addResultsForCases: SUCCESS - the request data is ${JSON.stringify(data)}`);
+			output.log(`addResultsForCases: SUCCESS - response data is ${JSON.stringify(res.data)}`);
 			return res.data;
 		}).catch(error => {
 			const parsedError = error && error.response && error.response.data ? error.response.data.error : error;
-			output.error(`Cannot add result for case due to ${parsedError}`);
-			output.error(`Request data was: ${JSON.stringify(data)}`);
+			output.error(`addResultsForCases: ERROR - cannot add result for case due to ${parsedError}`);
+			output.error(`addResultsForCases: ERROR - request data was ${JSON.stringify(data)}`);
 		});
 	}
 
 	async addAttachmentToResult(resultId, imageFile) {
 		let form = new FormData();
 		form.append('attachment', fs.createReadStream(path.join(global.output_dir, imageFile.toString())));
 
-		await this.axios({
+		this.axios({
```

Comment:
I'm not sure we still need await here?

Reply:
oups, not sure how that happened. Wrong copy/paste possibly